src/neo/Network/P2P/LocalNode.cs

```diff
@@ -127,6 +127,30 @@ internal static IPEndPoint GetIpEndPoint(string hostAndPort)
             return null;
         }
 
+        /// <summary>
+        /// Check the new connection <br/>
+        /// If it is equal to the Nonce of local or any remote node, it'll return false, else we'll return true and update the Listener address of the connected remote node.
+        /// </summary>
+        /// <param name="actor">Remote node actor</param>
+        /// <param name="node">Remote node</param>
+        public bool AllowNewConnection(IActorRef actor, RemoteNode node)
+        {
+            if (node.Remote is null) return false;
```

Comment:
This cannot be `null` because it is assigned in the constructor.

Reply:
Yes, it can't be null now.